distribution/std-lib/Standard/src/Geo/Geo_Json.enso

```diff
@@ -0,0 +1,42 @@
+from Standard.Base import all
+
+import Standard.Geo.Geo_Json.Internal
+import Standard.Table.Data.Table
+
+## Convert a GeoJSON document into a dataframe.
```

Comment:
As far as I understand it this is a big misunderstanding of what is necessary. The idea was to perform format inference in standard JSON's `to_table`, and if the JSON follows the geo-json format it converts it into a table as geo-json. This code itself is good, but we shouldn't have to have a conflicting extension method in scope (as this will break things).

Reply:
Yes, that is true. Sorry if I caused any confusion here.